tutorials/onchain-governance/fast-track-gov-proposal.md

```diff
@@ -0,0 +1,220 @@
+
+!!!note
+    In this tutorial, the focus will be on the main steps and core logic within the main function. For clarity and conciseness, the implementation details of individual functions will be available in expandable tabs below each section. At the end of the tutorial, you'll find the complete code for reference.
+
+### Spin Up the Polkadot Fork
+
+Before you can interact with the forked network, you need to start it using Chopsticks. Open a new terminal window and run the following command:
+
+```
+npx @acala-network/chopsticks --config=polkadot
+```
+
+This command will start a local fork of the Polkadot network accesible at `ws://localhost:8000`. Keep this terminal window open and running throughout your testing process.
+Once your forked network is up and running, you can proceed with the following steps.
+
+### Set Up Dependencies and Structure
+
+Begin by adding the necessary imports and a basic structure:
+
+```typescript
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:imports'
+
+const main = async () => {
+	// The code will be added here
+
+	process.exit(0)
+}
+
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:try-catch-block'
+```
+
+This structure provides the foundation for your script. It imports all the necessary dependencies and sets up a main function that will contain the core logic of your proposal submission process.
+
+### Connect to the Forked Chain
+
+Inside your `main` function, add the code to connect to your local Polkadot fork:
+
+```typescript hl_lines="2-3"
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:main'
+```
+
+???+ function "**connectToFork** ()"
+
+	```typescript
+	--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:connectToFork'
+	```
+
+### Create and Submit the Proposal
+
+In this step, you will perform the following actions:
+
+1. Define the call you want to execute and its origin
+2. Create a preimage using the selected call
+3. Submit the proposal. It uses the preimage hash (obtained from the call) as part of the proposal submission process. The proposal is submitted with the selected origin
+4. Place decision deposit. This deposit is required to move the referendum from the preparing phase to the deciding phase
+
+```typescript hl_lines="5-14"
+--8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/test-proposal.ts:main'
+```
+
+!!!note
+    The `setCodeWithoutChecks` extrinsic used in this example is for demonstration purposes only. Replace it with the specific extrinsic that matches your governance proposal's intended functionality. Ensure the call matches the runtime requirements and governance process of your target Polkadot SDK-based network.
```

Comment:
nit picking here, but would be cool to add a link to https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/struct.Pallet.html#method.set_code_without_checks for example

However as the method used here is `setCodeWithoutChecks` and the reference is `set_code_without_checks`, maybe you can briefly explain that this js snippet is executing that call on the pallet side